common/djangoapps/student/tests/test_verification_status.py

```diff
@@ -131,6 +133,17 @@ def test_fully_verified(self, expiration):
         response = self.client.get(self.dashboard_url)
         self.assertContains(response, attempt.expiration_datetime.strftime("%m/%d/%Y"))
 
+    @patch("lms.djangoapps.verify_student.services.is_verification_expiring_soon")
+    def test_verify_resubmit_button_on_dashboard(self, mock_expiry):
+        mock_expiry.return_value = True
+        SoftwareSecurePhotoVerification.objects.create(user=self.user, status='approved', expiry_date=now())
```

Comment:
I believe `expiry_date` should be in the period which we use to evaluate expiring soon, setting it to now means it's already expired.